src/Checkbox/Checkbox.stories.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+import {Meta} from '@storybook/react'
+import {Box, Checkbox, CheckboxProps} from '..'
+import FormControl from '../FormControl'
+import {
+  FormControlArgs,
+  formControlArgTypesWithoutValidation,
+  getFormControlArgsByChildComponent,
+} from '../utils/form-story-helpers'
+
+const excludedControlKeys = ['required', 'value', 'validationStatus', 'sx']
+
+export default {
+  title: 'Components/Checkbox',
+  component: Checkbox,
+  parameters: {controls: {exclude: excludedControlKeys}},
+} as Meta
+
+export const Playground = ({value: _value, checked, ...args}: FormControlArgs<CheckboxProps>) => {
+  const {parentArgs, labelArgs, captionArgs} = getFormControlArgsByChildComponent(args)
+
+  return (
+    <Box as="form">
+      <FormControl {...parentArgs}>
+        <Checkbox value="default" checked={checked} {...args} />
+        <FormControl.Label {...labelArgs} />
+        {captionArgs.children && <FormControl.Caption {...captionArgs} />}
+      </FormControl>
+    </Box>
+  )
+}
+Playground.args = {
+  checked: false,
+  indeterminate: false,
+  visuallyHidden: false,
+  disabled: false,
+  labelChildren: 'Default checkbox',
+  captionChildren: 'Always unchecked unless `checked` is set to true in Storybook controls',
```

Comment:
Non-blocking nitpick: maybe write the property keys for children like this so they don't look like prop names:
```
['label children']: 'Default checkbox'
```

This would apply to `labelChildren`, `captionChildren`, and `validationChildren`.

Reply:
I really like this idea, but I couldn't get it to work 🤔 maybe it needs to happen also in the form-story-helpers.. I think I would prefer to follow up on this after I finish all of the form stories

Reply:
Aw man that's too bad. Yea, we can absolutely come back to this later if it ends up being a problem.